.github/workflows/release.yaml

```
@@ -15,6 +15,11 @@ on:
         description: 'Schema Patch version'
         type: string
         required: true
+      skip-stack-release:
+        description: 'Skip stack release'
```

Nice to have this option, thanks! Could you make it a little clearer in the description: 
```suggestion
        description: 'Skip stack release: skip pushing the newly released schema to the downstream repositories'
```

---

Thanks, updated